example/package.json

```diff
@@ -11,7 +11,7 @@
     "format": "prettier App.tsx -w"
   },
   "dependencies": {
-    "@ionic/portals-react-native": "^0.0.1",
+    "@ionic/portals-react-native": "file:../ionic-portals-react-native-0.0.1.tgz",
```

Comment:
Was this intended to replace with a local zip

Reply:
My intention is to reconfigure the project to have a monorepo structure where the example project is not a subfolder of the library itself